gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/ApplicationServiceImpl.java

```diff
@@ -58,6 +58,7 @@
 import io.gravitee.rest.api.service.impl.configuration.application.registration.client.register.ClientRegistrationResponse;
 import io.gravitee.rest.api.service.notification.ApplicationHook;
 import io.gravitee.rest.api.service.notification.HookScope;
+import io.micrometer.common.util.StringUtils;
```

Comment:
It's ok but I just find it strange to use the micrometer one. I'm used to use `import org.apache.commons.lang3.StringUtils;`